taxi_service/settings.py

```diff
@@ -24,7 +24,7 @@
              "-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
 ALLOWED_HOSTS = []
```

Comment:
You have an empty ALLOWED_HOSTS list. This is a security risk, as it allows requests to be sent from any host. You should specify the hosts that are allowed to connect to your Django application.